frontend/public/components/utils/status.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
```

Comment:
We should probably give this file a more specific name since it's only service catalog instance and binding status. service-catalog-status.tsx?